gtfs-realtime/spec/en/reference.md

```diff
@@ -297,6 +301,20 @@ For describing passenger occupancy levels on a linear scale, see `occupancy_perc
 | _**NO_DATA_AVAILABLE**_ | _The vehicle or carriage doesn't have any occupancy data available at that time._ |
 | _**NOT_BOARDABLE**_ | _The vehicle or carriage is not boardable and never accepts passengers. Useful for special vehicles or carriages (engine, maintenance carriage, etc…)._ |
 
+## _enum_ CrowdLevel
+
+The passenger crowd level for the vehicle or carriage. CrowdLevel enum values must be interpreted on an ordinal scale in increasing order from _**NOT_CROWDED**_ to _**CROWDED**_. 
```

Comment:
I would add another sentence saying "In other words, producers must represent the minimum crowdedness value with _**NOT_CROWDED**_ and the maximum crowdedness value with _**CROWDED**_."

There are going to be questions as to when `CrowdLevel` should be used and when `OccupancyStatus` should be used. I'd be inclined to provide an explicit example to make sure there is no confusion.

Reply:
Also, should producers take operational considerations like social distancing into account when publishing `CrowdLevel`? There is a line in `occupancy_percentage` definition about this - I'd suggest a similar mention here.

Reply:
As a producer, we prefer not to change the crowd level that we publish according to social distancing rules. We prefer to "tell it as it is" and let the client decide wether he is confortable or not with crowding in the vehicle. (since some clients are more nervous than others)

Reply:
Please see my latest commit where I added the sentence "or as otherwise defined by current operating regulations". If not otherwise defined, I think there won't be any introduced ambiguity. Thoughts?